expenses-manager-frontend/src/main/frontend/src/app/daily-stats/daily-stats.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="dashboard-container">
+    <div class="chart-container">
```

Comment:
De asemenea aici va trebui sa avem un date picker iar utilizatorul cand da click pe o zi, sa se faca call catre back-end si sa se aduca cheltuilelile pe ziua respectiva. Dar acum partea aceea de backend este in progress sugerez sa facem momentan doar FE si ulterior facem si legatura FE-BE.